octodns_rackspace/__init__.py

```diff
@@ -47,17 +47,25 @@ class RackspaceProvider(BaseProvider):
     SUPPORTS = set(
         ('A', 'AAAA', 'ALIAS', 'CNAME', 'MX', 'NS', 'PTR', 'SPF', 'TXT')
     )
-    TIMEOUT = 5
 
     def __init__(
-        self, id, username, api_key, ratelimit_delay=0.0, *args, **kwargs
+        self,
+        id,
+        username,
+        api_key,
+        ratelimit_delay=0.0,
+        timeout=5,
```

Comment:
Let's go ahead and bump this up to 10s while you're in here.  There's no specific reason to have 5s, it's just important to have something, so 10s is as good as anything and if you've hit problems it makes sense to me to bump.

```suggestion
        timeout=10,
```

Reply:
Nice, I in fact have 10s in my config file, but I left it just in case there was any reason for the 5s.